docs/production.md

```diff
@@ -80,8 +79,9 @@ system, exposes the same API as a single Docker host, which means you can use
 Compose against a Swarm instance and run your apps across multiple hosts.
 
 Compose/Swarm integration is still in the experimental stage, and Swarm is still
-in beta, but if you'd like to explore and experiment, check out the
-[integration guide](https://github.com/docker/compose/blob/master/SWARM.md).
+in beta, but if you'd like to explore and experiment, check out the <a
+href="https://github.com/docker/compose/blob/master/SWARM.md">integration
+guide</a>.
```

Comment:
What's the reason for changing these to html links instead of the markdown? 

Does some part of the build process attempt to convert the extension to `.html` ?